Tasks/AzureFunctionAppV1/deploymentProvider/ConsumptionWebAppDeploymentProvider.ts

```diff
@@ -30,6 +38,83 @@ export class ConsumptionWebAppDeploymentProvider extends AzureRmWebAppDeployment
         }
         return storageData;
     }
+
+    private async uploadPackage(storageDetails, deployPackage) : Promise<string> {
+        let defer = Q.defer<string>();
+        let storageAccount = storageDetails["AccountName"];
+        let storageKey = storageDetails["AccountKey"];
+        const blobService = azureStorage.createBlobService(storageAccount, storageKey);
+
+        const containerName: string = 'azure-pipelines-deploy';
+        const blobName: string = `package_${Date.now()}.zip`;
```

Comment:
Can this vary based on agent locale and will it continue to have supported character set for blob name?

Reply:
`Date.now()` would return a timestamp of type `int`. we should be good here.